src/store/importer.js

```diff
@@ -16,7 +16,7 @@ export default function importer(state = null, action) {
     else if (action.type == types.EXECUTE_IMPORT + '_FULFILLED') {
         return Object.assign({}, state, {
             importIsPending: false,
-            importStats: action.payload.data.data.report,
+            importStats: action.payload.data.data,
```

Comment:
Since this no longer contains just stats (the "statistics" of the import), perhaps this should be renamed, along with the `stats` local variable in `ImportPane`. I'm thinking something along the lines of `importResponse` or `lastImportLog`.